CleverTapSDK/InApps/CustomTemplates/CTAppFunctionBuilder.m

```diff
@@ -14,7 +14,12 @@
 @implementation CTAppFunctionBuilder
 
 - (nonnull instancetype)initWithIsVisual:(BOOL)isVisual {
-    self = [super initWithType:FUNCTION_TYPE isVisual:isVisual];
+    return [self initWithType:FUNCTION_TYPE isVisual:isVisual isSystemDefined:NO];
```

Comment:
Reuse the constructor `initWithIsVisual:isSystemDefined:`.

Reply:
Done